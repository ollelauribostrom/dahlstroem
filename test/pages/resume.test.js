import React from 'react';
import { shallow } from 'enzyme';

import ResumePage from '../../src/pages/resume';

const mockData = {
  allDatoCmsGrant: {
    edges: [
      { node: { description: 'Grant1' } },
      { node: { description: 'Grant2', href: 'http://google.com' } },
    ],
  },
  allDatoCmsExhibition: {
    edges: [
      { node: { description: 'Exhibition1' } },
      { node: { description: 'Exhibition2', href: 'http://google.com' } },
    ],
  },
  allDatoCmsEducation: {
    edges: [
      { node: { description: 'Education 1' } },
      { node: { description: 'Education 2' } },
    ],
  },
};

describe('Tests for Resume page', () => {
  it('renders correctly', () => {
    const wrapper = shallow(
      <ResumePage location={{ pathname: '/test' }} data={mockData} />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
