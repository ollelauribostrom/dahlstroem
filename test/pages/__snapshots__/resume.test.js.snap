// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for Resume page renders correctly 1`] = `
<Layout
  color={
    Object {
      "hex": "#000",
    }
  }
  menuBackground={
    Object {
      "hex": "#000",
    }
  }
  menuColor={
    Object {
      "hex": "#fff",
    }
  }
  path="/test"
  width={810}
>
  <SEO
    keywords={
      Array [
        "resume",
        "cv",
        "education",
        "exhibitions",
        "fellowships",
        "grants",
        "residencies",
      ]
    }
    lang="en"
    meta={Array []}
    title="Resume"
  />
  <Section
    title="Education"
  >
    <SectionItem
      description="Education 1"
      key="education-0"
    />
    <SectionItem
      description="Education 2"
      key="education-1"
    />
  </Section>
  <Section
    title="Exhibitions"
  >
    <SectionItem
      description="Exhibition1"
      key="exhibition-0"
    />
    <SectionItem
      description="Exhibition2"
      href="http://google.com"
      key="exhibition-1"
    />
  </Section>
  <Section
    title="Fellowships, Grants, Residencies"
  >
    <SectionItem
      description="Grant1"
      key="grant-0"
    />
    <SectionItem
      description="Grant2"
      href="http://google.com"
      key="grant-1"
    />
  </Section>
</Layout>
`;
